sklearn/linear_model/perceptron.py

```diff
@@ -123,6 +123,19 @@ class Perceptron(BaseSGDClassifier):
     ``Perceptron()`` is equivalent to `SGDClassifier(loss="perceptron",
     eta0=1, learning_rate="constant", penalty=None)`.
 
+    Examples
+    --------
+    >>> from sklearn.datasets import load_digits
+    >>> from sklearn.linear_model import Perceptron
+    >>> X, y = load_digits(return_X_y=True)
+    >>> clf = Perceptron(random_state=0)
```

Comment:
``clf = Perceptron(tol=1e-3, random_state=0)``
You get FutureWarning because we're changing the API here.